GDJS/Runtime/runtimegame.js

```diff
@@ -60,6 +60,10 @@ gdjs.RuntimeGame = function(data, options) {
   this._options = options;
 
   //Optional client to connect to a debugger:
+  /** 
+   * @type {?gdjs.IDebuggerClient} 
```

Comment:
Is this type necessary? If not maybe we can skip it to avoid polluting too much the code.
private is also automatically added for JSDoc because of the underscore (not for typescript, but this creates problem in newer version of typescript that does not understand that class methods using `prototype` are allowed to access private variables).

Reply:
IMO the more typing the better.